docker/test/stateful_with_coverage/s3downloader

```diff
@@ -24,25 +27,35 @@ def build_url(base_url, dataset):
 
 def dowload_with_progress(url, path):
     logging.info("Downloading from %s to temp path %s", url, path)
-    with open(path, 'w') as f:
-        response = requests.get(url, stream=True)
-        response.raise_for_status()
-        total_length = response.headers.get('content-length')
-        if total_length is None or int(total_length) == 0:
-            logging.info("No content-length, will download file without progress")
-            f.write(response.content)
-        else:
-            dl = 0
-            total_length = int(total_length)
-            logging.info("Content length is %ld bytes", total_length)
-            for data in response.iter_content(chunk_size=4096):
-                dl += len(data)
-                f.write(data)
-                if sys.stdout.isatty():
-                    done = int(50 * dl / total_length)
-                    percent = int(100 * float(dl) / total_length)
-                    sys.stdout.write("\r[{}{}] {}%".format('=' * done, ' ' * (50-done), percent))
-                    sys.stdout.flush()
+    for i in range(RETRIES_COUNT):
```

Comment:
unfortunately, docker doesn't support symlinks and require all files to be in `Dockerfile's` directory. So we have to support to copies :(